tlsobs/main.go

```diff
@@ -118,6 +118,10 @@ getresults:
 		if err != nil {
 			panic(err)
 		}
+		if results.Complperc == 100 && results.ScanError != "" {
+			fmt.Printf("Scan failed with error: %s\n", results.ScanError)
+			os.Exit(81)
```

Comment:
Should this be documented in the README?
Since we are differentiating exit codes ( so not considering this a web page fault ) we could document it in case anybody is using tlsobs as part of CI/Sec scan.

Reply:
Probably, but honestly I'd only bother if someone asks for it. Otherwise, it's easy enough to read the source.

Reply:
Sounds reasonable.
Everything looks good to merge!